<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntCenterMemberWidget</class>
 <widget class="QWidget" name="EntCenterMemberWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FinanMemberWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EntCenterMemberWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QPushButton" name="addMemberButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>590</y>
     <width>88</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>添加成员</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE;}
QPushButton:hover,pressed{background-color:#F4F4F4;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;}</string>
   </property>
   <property name="text">
    <string>添加成员</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAudit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>600</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>15</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>成员审核</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: #1E8BED;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>成员审核</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>20</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>20</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>45</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>职务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>操作</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>730</width>
     <height>540</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>740</width>
     <height>540</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{border-bottom-left-radius:0px;}
QListWidget::Item{background:rgb(255,255,255);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>28</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>角色</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
